scripts/trento-support.sh

```diff
@@ -58,8 +58,8 @@ collect_kubernetes_state() {
     kubectl get pods
 
     echo "#==[ Command ]======================================#"
-    echo "# $(which kubectl) logs deploy/trento-server-runner"
-    kubectl logs deploy/trento-server-runner
+    echo "# $(which kubectl) logs deploy/trento-server-wanda"
+    kubectl logs deploy/trento-server-wanda
```

Comment:
Wanda has the `init` container as well. We should take logs from both, as we do with the web. So we should add a new logs line